quartz/plugins/transformers/frontmatter.ts

```diff
@@ -40,11 +43,26 @@ function coerceToArray(input: string | string[]): string[] | undefined {
     .map((tag: string | number) => tag.toString())
 }
 
+function getAliasSlugs(aliases: string[], { argv }: BuildCtx, file: VFile): FullSlug[] {
+  const dir = path.posix.relative(argv.directory, path.dirname(file.data.filePath!))
+  const slugs: FullSlug[] = aliases.map(
+    (alias) => path.posix.join(dir, sluggify(alias)) as FullSlug,
```

Comment:
```suggestion
    (alias) => path.posix.join(dir, sluggifyFilePath(alias as FilePath, false)) as FullSlug,
```

albeit some type coercion.